packages/ingest/package.json

```diff
@@ -16,7 +16,8 @@
   "ava": {
     "files": [
       "!test/fixtures/**/*"
-    ]
+    ],
+    "verbose": true
```

Comment:
Do we still need/want this?

Reply:
I like verbose because it makes the output better when running `ava --watch`. I'll revert it and just use `ava -wv` going forward.

Reply:
It doesn't seem like the change was reverted here or elsewhere? To be clear, I have no issue with the change per se, I was just wondering